src/main/data/facilities.json

```diff
@@ -2704,7 +2718,8 @@
   },
   {
     "code": "OND",
-    "description": "old/new cineproductions"
+    "description": "old/new cineproductions",
+    "url": "www.oldnew.de"
```

Comment:
Should we always have a scheme? I think this should be:

> https://oldnew.de

Reply:
I left this for ideas for validation, as noted next comment.

Reply:
http:// or https:// validates on the uri type, so yes.